arch/xtensa/core/mmu.c

```diff
@@ -156,7 +166,7 @@ void xtensa_init_paging(uint32_t *l1_page)
 			 "isync\n"
 			 "iitlb %8\n" /* invalidate pc */
 			 "isync\n" /* <--- traps a ITLB miss */
-			 :: "r"(regs.ptevaddr), "r"(regs.rasid),
+			 :: "r"(regs.ptevaddr), "r"(initial_rasid),
```

Comment:
Missed this before merge, but a nitpick might have been that it would be cleaner to initialize regs correctly rather than treating this one register specially.